src/Tizen.Applications.DataControl/Interop/Interop.DataControl.cs

```diff
@@ -346,8 +346,8 @@ internal struct SqlRequestCallbacks
         [DllImport(Libraries.DataControl, EntryPoint = "datacontrol_provider_write_socket")]
         internal static extern unsafe ResultType WriteSelectResult(int socketFd, byte* buffer, uint nbytes, out uint bytesWrite);
 
-        [DllImport(Libraries.DataControl, EntryPoint = "data_control_provider_send_data_change_noti")]
-        internal static extern ResultType SendDataChange(IntPtr handle, ChangeType type, SafeBundleHandle data);
+        [DllImport(Libraries.DataControl, EntryPoint = "data_control_provider_send_data_change_noti_by_data_id")]
+        internal static extern ResultType SendDataChange(string dataId, ChangeType type, SafeBundleHandle data);
```

Comment:
EXPORT_API int data_control_provider_send_data_change_noti_by_data_id(
		char *data_id, data_control_data_change_type_e type, bundle *data)

Native API signature is worng,
``` char *data_id ``` it should be ``` const char *data_id```

Reply:
I updated it
https://review.tizen.org/gerrit/#/c/platform/core/appfw/data-control/+/193370/